provider/ecs/ecr.go

```diff
@@ -18,9 +18,13 @@ type ECR struct {
 
 // Creates ECR repository
 func (e *ECR) CreateRepository() error {
+	scanOnPush := true
 	svc := ecr.New(session.New())
 	input := &ecr.CreateRepositoryInput{
 		RepositoryName: aws.String(e.RepositoryName),
+		ImageScanningConfiguration: &ecr.ImageScanningConfiguration{
+			ScanOnPush: &scanOnPush,
```

Comment:
you can use aws.Bool(scanOnPush) instead of &scanOnPush. Is nicer, as we're using that already in the code a lot. See  aws.String() for example